src/main/java/hunternif/mc/atlas/client/BiomeTextureConfig.java

```diff
@@ -41,8 +42,41 @@ protected void loadData(JsonObject json, BiomeTextureMap data, int version) {
 					+ " disregarding this config entirely");
 			return;
 		}
+
 		for (Entry<String, JsonElement> entry : json.entrySet()) {
-			int biomeID = Integer.parseInt(entry.getKey());
+			String key = entry.getKey();
+
+			if (!(key.length() > 0)) {
+				Log.warn("No biome ID specified, skipping entry");
+				return;
+			}
+
+			int biomeID;
+
+			try {
+				// See if the biome id is an integer
+				biomeID = Integer.parseInt(key);
+			} catch(NumberFormatException e) {
+				// If it is not an integer, attempt to find the biome ID assuming it is a resource location
+				ResourceLocation biomeResourceLocation;
+
+				if (key.contains(":")) {
+					String[] biomeStringSplit = key.split(":");
+					biomeResourceLocation = new ResourceLocation(biomeStringSplit[0], biomeStringSplit[1]);
+				} else {
+					biomeResourceLocation = new ResourceLocation(key);
+				}
+
+				Biome biome = Biome.REGISTRY.getObject(biomeResourceLocation);

```

Comment:
```java
Biome biome = ForgeRegistries.BIOMES.getValue(new ResourceLocation(key));
```

Use the Forge provided classes rather than the Vanilla ones.

Everything else above this is unnecessary.